src/doc/src/guide/continuous-integration.md

```diff
@@ -1,25 +1,8 @@
 ## Continuous Integration
 
-### Travis CI
+### Getting Started
```

Comment:
Is this header here really needed? And also, in general maybe the header levels have to be adjusted to be semantically more useful?

Currently the header hierarchy looks like this:
- Continuous Integration
  - Getting Started
  - GitHub Actions
  - GitLab CI
  - builds.sr.ht
  - Verifying Latest Dependencies

Maybe something like this would be better?
- Continuous Integration
  - _\<no sure how to name this\>_
    - GitHub Actions
    - GitLab CI
    - builds.sr.ht
  - Verifying Latest Dependencies